spirv_reflect.h

```diff
@@ -1479,6 +1479,10 @@ class ShaderModule {
   const char*                   GetEntryPointName(uint32_t index) const;
   SpvReflectShaderStageFlagBits GetEntryPointShaderStage(uint32_t index) const;
 
+  SpvReflectEntryPoint::LocalSize GetEntryPointLocalSize(uint32_t index) const {
```

Comment:
It would be preferable here not to return a struct by copy - especially since it's an embeeded struct. 

Instead could it return `x`, `y`, `z` by pointer parameter and `index` is ranged check tgo make sure it's less than `SpvReflectShaderModule ::entry_point_count` and return `SPV_REFLECT_RESULT_ERROR_ELEMENT_NOT_FOUND` otherwise?